aws/SDK_CHANGELOG.md

```diff
@@ -1,6 +1,56 @@
 vNext (Month Day, Year)
 =======================
 
+**Breaking Changes**
+
+Several breaking changes around `aws_smithy_types::Instant` were introduced by smithy-rs#TODO:
```

Comment:
can you:
- list out the crates with breaking changes as a top level item (for ease of release cutting)
- add the changes to aws-sigv4 and aws-sig-auth

Reply:
> list out the crates with breaking changes as a top level item (for ease of release cutting)

I'm not sure I understand how this makes releasing eaiser.

Reply:
Added a changelog entry for `aws-sigv4`. There weren't any breaking changes to `aws-sig-auth`.